.pipelines/npm/npm-conformance-tests.yaml

```diff
@@ -72,8 +72,8 @@ jobs:
   - script: |
       echo Tag: $(TAG)
       echo ResourceGroup: $(RESOURCE_GROUP)
+      docker run --privileged --rm tonistiigi/binfmt --install arm64
```

Comment:
is this installing arm64 emu support on the host that's executing this pipeline step? that doesn't feel right

Reply:
I believe it is, I ran into a bug otherwise, here's the thread I followed
https://github.com/moby/buildkit/issues/1929#issuecomment-758269704

Reply:
i mean, if it works, it works, but it doesn't seem right to install things on the host during a pipeline run

Reply:
Mat's oof on leave, I will check with him once he's back on how to get this installed on the images pre-pipeline, in the meantime, I'll resolve this though to get our arm64 images to AKS